### PR TITLE
Changing brands select into boostrap dropdown

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | The select of brand module on category page is ugly, let's use a bootstrap dropdown we already use on this page
| Type?         | improvement
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #9670
| How to test?  | Install the ps_brandlist module, then change the configuration to dropdown instead of plain text

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16524)

### CODE REVIEW

File: views/templates/_partials/brand_form.tpl [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
+        href="{$brand['link']}"
+        class="select-list dropdown-item"
+      >
+        {$brand['name']}
+      </a>
     {/foreach}
-  </select>
-</form>
+  </div>
+</div>
```
Comment (PierreRambaud) [inline views/templates/_partials/brand_form.tpl]:
```suggestion
</div>

```

Comment (NeOMakinG) [reply views/templates/_partials/brand_form.tpl]: Pas compris, l'indent est bonne et le nombre de div egalement

Comment (PierreRambaud) [reply views/templates/_partials/brand_form.tpl]: image
This mean there is no newline at the end of the file which is mandatory.

File: views/templates/_partials/brand_form.tpl [hunk 2]
```diff
@@ -1,8 +1,22 @@
...
-    <option value="">{l s='All brands' d='Modules.Brandlist.Shop'}</option>
+<div class="brands-sort dropdown">
+  <button
+    class="btn-unstyle select-title"
+    rel="nofollow"
+    data-toggle="dropdown"
+    aria-haspopup="true"
+    aria-expanded="false">
+    {l s='All brands' d='Modules.Brandlist.Shop'}
+    <i class="material-icons float-xs-right">&#xE5C5;</i>
```
Comment (PierreRambaud) [inline views/templates/_partials/brand_form.tpl]:
```suggestion
    <i class="material-icons float-xs-right">arrow_drop_down</i>
```

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**